openssh/README.md

```diff
@@ -0,0 +1,106 @@
+## Purpose 
+
+This directory contains a Dockerfile that builds the [OQS OpenSSH fork](https://github.com/open-quantum-safe/openssh), which allows to establish a quantum-safe SSH connection using quantum-safe keys and quantum-safe authentication.
+
+## Quick start
+
+[Install Docker](https://docs.docker.com/install) and run the following commands in this directory:
+
+1. Run `docker build -t oqs-openssh-img .` This will generate the image with the name `oqs-openssh-img`
+2. `docker run --name oqs-openssh-server -ditp 2222:2222 --rm oqs-openssh-img`
+This will start a docker container that has sshd listening for SSH connections on port 2222 and this port is forwarded to and accessible via `localhost:2222`.
+3. `docker run --rm --name oqs-openssh-client -dit oqs-openssh-img` will start a docker container with the same properties as `oqs-openssh-server` except the port 2222 is not published.
```

Comment:
Would it be better to leave away the numbers 3 and 4 (and rephrase somewhat), as these are an alternative (3) and an optional subsequent step (4), but not essentially required to run through the demo?